packages/client/src/editor/render.ts

```diff
@@ -7,8 +7,7 @@ import { numberToCoord } from '../utils';
 
 const reset = (ctx: CanvasRenderingContext2D, canvas: HTMLCanvasElement) => {
   ctx.setTransform(1, 0, 0, 1, 0, 0);
-  ctx.fillStyle = '#FFFFFF';
```

Comment:
`ctx.clearRect()` sets all pixels in canvas to transparent black (which is rgba(0,0,0,0) or something like that)

fillStyle is only needed for fillRect, however the change to clearRect was needed in order for the drawing canvas to be transparent so that images can be displayed